packages/abi-typegen/test/fixtures/templates/predicate-with-configurable/factory.hbs

```diff
@@ -78,7 +78,7 @@ const _abi = {
         "type": 2,
         "typeArguments": null
       },
-      "offset": 560
+      "offset": 1152
```

Comment:
How much bigger are the contract sizes? jw as the offset has doubled 😬

Reply:
@danielbate How can I verify that?

Reply:
Before/after the file size of the bin file. Tbh we can't do anything about this so this won't be a blocker.

Reply:
@danielbate Based on the example that I picked, it appears they've increased ~ 18%